### PR TITLE
Built CRUD routes and wrote README file

### PR DESCRIPTION
**Pull request by tonyperletti**

### CODE REVIEW

File: server/index.js [hunk 1]
```diff
@@ -50,7 +50,45 @@ app.patch('/api/reviews', function(req, res) {
...
+
+app.post('/api/reviews', (req, res) => {
+  // console.log(req.body);
+  const review = req.body;
+  Review.create(review)
+  .then(() => {
+    res.send('successfully posted review');
+  })
+  .catch(error => console.log(error))
+});
```
Comment (PeanutJack) [inline server/index.js]: The readme says the post route uses the id in URL but I don't see it being used here.

File: server/index.js [hunk 2]
```diff
@@ -50,7 +50,45 @@ app.patch('/api/reviews', function(req, res) {
...
 
+app.put('/api/reviews', (req, res) => {
+  // console.log(req.query.id);
+  const id = req.query.id;
+  const body = req.body;
+  Review.update({ id }, body)
+  .then(() => {
+    res.send('successfully updated');
+  })
+  .catch(error => console.log(error))
```
Comment (PeanutJack) [inline server/index.js]: I think you might be missing a semicolon after the catch block, and I think the chains of a promise should be indented further.

**Review by PeanutJack — COMMENTED** [review]
One minor readme inaccuracy and some possible linting issues. Nothing major enough to stop you from merging if you want, but there is also room to clean if you'd like.

File: server/index.js (hunk 1 above)
Comment (tonyperletti) [reply server/index.js]: Thanks, I forgot to complete that part of the post.

**Pull request merged**